azure-daily.yml

```diff
@@ -0,0 +1,38 @@
+# Azure Pipelines file, see https://aka.ms/yaml
+
+# scheduled triggers are not supported in pipeline syntax yet, so it had to be
+# set up from Azure UI
+trigger: none
+
+jobs:
+  - job: Daily Docs
+    timeoutInMinutes: 360
+    pool:
+      name: 'linux-pool'
+    checkout:
+    steps:
+      - checkout: self
+      - bash: ci/dev-env-install.sh
+        displayName: 'Build/Install the Developer Environment'
+      - bash: ci/configure-bazel.sh
+        displayName: 'Configure Bazel'
+      - bash: |
+        set -euo pipefail
+        export GOOGLE_APPLICATION_CREDENTIALS=$(mktemp)
+        echo "$GOOGLE_APPLICATION_CREDENTIALS_CONTENT" > $GOOGLE_APPLICATION_CREDENTIALS
+        LATEST_RELEASE=$(curl https://api.github.com/repos/digital-asset/daml/releases -s | jq -r '. | map(select(.prerelease == false)) | map(.tag_name)[0]')
+        git checkout $LATEST_RELEASE
+        bazel build //docs
+        cp bazel-genfiles/docs/html.tar.gz $(Build.StagingDirectory)/
+        cd $(Build.StagingDirectory)
+        tar xzf html.tar.gz
+        gsutil rsync -d -r . gs://daml-docs
```

Comment:
I don’t know anything about `gsutil` so this could be totally wrong but this looks like you might end up uploading the tarball as well. You might want to extract it to a different directory and only upload the files in that directory.

Reply:
Good catch! That dot should actually read `html` as what we want to upload are the extracted contents of the tarball.